tests/AbstractServiceTest.php

```diff
@@ -40,7 +40,7 @@ protected function setUpMapper($mapperName, $entityBuilderMethod)
     {
         $this->mockMapper = $this->createMock($this::MAPPER_NS . $mapperName);
 
-        $this->mockMapper->expects($this->any())
+        $this->mockMapper
```

Comment:
Just checked. is not consistent the amount of calls to getDomainModel. So Expects($this->once()) breaks many tests. jeje. I will manage anyway...maybe I can do something as I go with the tidy up.